tailwind.config.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
   theme: {
     extend: {
       colors: {
+        overlayColor:'#01010180',
```

Comment:
Looks great! make sure to replace the hex value `bg-[#0101080]` in our two modal components with our new custom color value `bg-overlayColor` then we'll be off to the races!